### PR TITLE
adding my art to list

### PR DESCRIPTION
**Pull request by Pariskarpoudel**
plz addme

### CODE REVIEW

**Comment by Pariskarpoudel** [comment]
adddddddd

File: include.js [hunk 1]
```diff
@@ -2361,6 +2361,13 @@ let cards = [
     author: 'mohamedmosalam', 
     githubLink: 'https://github.com/MohamedAboMousallam' 
   },
+  {
+    artName: 'CSS Duck art', // change this to the name of your artwork
+    pageLink: './Art/animatron/index.html', // change this
+    imageLink: './Art/animatron/segment-display.gif', // change this
+    author: 'pari',// use your name
+    githubLink: 'https://github.com/Pariskarpoudel' // change this
```
Comment (sophiabrandt) [inline include.js:2369]: Please remove the comments and adjust your entry with the correct details (see other comment).

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please move your artwork into a folder that has the same name as your Github username.

For details, see the instructions in the README file.

**Comment by LaurelineP** [comment]
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedbacks

# Feedbacks
- [ ] your contribution folder is not named as your github username
- [ ] gif file is empty - please provide a valid gif file

## Acceptance criteria for a contribution
- [ ] the contribution includes a new folder named after the author github user name  in `./Art`
- [x] the contribution includes an HTML file
    - [x] no CSS
    - [x] no JS
    - [x] no embedded  SVG 
    - [x] has a `<link>` tag to the CSS file
- [x] the contribution includes a CSS file
- [ ] the contribution includes the animated file (gif) or picture (jpeg, jpg, png )
    _the pictural representation must not include the browser frame, just the animation_
    Your gif file has no content ( you can check that on the PR' changes tab )
- [x] the contribution includes the correct contribution's details in `include.js`

Please address the above feedbacks and/or unchecked acceptance criteria - if you need guidance , please refer to the `README.md` file.

**Pull request closed without merging**